### DISCUSSION TITLE
Currently playing layout

### DISCUSSION THREAD

**Discussion by Toniob** [navidrome/navidrome#3239] [Ideas]
It will be great if there was a url to show the currently playing song. I mean, something really simple, just the cover for the song, its name and the artist, and that's all. The idea is to put this url into OBS and being able to show the currently playing song on a stream (twitch or youtube, or whatever).

**Comment by deluan** [comment]
Interesting idea. I never used OBS, do you have a sample (screenshot) of another music player that does this?

**Comment by Toniob** [comment]
You can see that here : https://www.twitch.tv/silent_genius/clip/GleamingBetterSquidLitty-QUjKzbifV-PjbIbX . At the top of the overlay, the song change.

Currently, my friend uses a tool called Snip : https://github.com/dlrudie/Snip with iTunes. But if we could have the same directly with Navidrome, it would be great.

**Comment by SickProdigy** [comment]
i'd have navidrome running on a screen and have obs select that screen and narrow down (crop) to the section you want to present in stream.

**Reply by Toniob** [reply]
It could be done like that, but that’s a workaround. It’s way easier to add a browser source in OBS. And with the browser source, you can change the CSS and the way the track is displayed. You can chose the colors, the disposition, … . How do you achieve that with the cropped screen capture ? And you cant’t move or resize your navidrome window or everything will break. It’s more flexible with the url IMHO.
